test_scripts/API/VehicleData/Refactor_Fuel_Information_Related_Vehicle_Data/008_App_version_is_more_then_version_fuelRange_in_DB_and_API.lua

```diff
@@ -0,0 +1,34 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0256-Refactor-Fuel-Information-Related-Vehicle-Data.md
+-- Description: The app is able to retrieve the parameter in case app version is more then parameter version
+-- parameter is listed in DB and API
+-- In case:
+-- 1) App is registered with syncMsgVersion=6.2
+-- 2) New params in `FuelRange` structure have since=6.0 and DB
```

Comment:
This script is not in scope for new proposal. FuelRange is old parameter, versioning should be checked only for new and deprecated parameters.